package com.pizzahut.dao;

import java.sql.Connection;
import java.sql.PreparedStatement;
import java.sql.SQLException;

import com.pizzahut.model.User;

public class Userdao {
	
	public void insert(User users) {
		ConnectionUtill con=new ConnectionUtill();
		String query="insert table users column(user_firstname,phonenumber,email,address,password) values(?,?,?,?,?)";
			Connection c=con.getDbconnection();
			PreparedStatement pstmt = null;
			try {
				pstmt = c.prepareStatement(query);
				pstmt.setString(2, users.getUsername());
				pstmt.setLong(3, users.getPhonenumber());
				pstmt.setString(4, users.getEmail());
				pstmt.setString(5, users.getAddress());
				pstmt.setString(6, users.getPassword());
				
			} catch (SQLException e) {
				// TODO Auto-generated catch block
				e.printStackTrace();
			}
}
}
